### PR TITLE
Add scripts for all commands, update readme/contributing.md

### PR DESCRIPTION
**Pull request by jacalata**
scripts not yet tested but at least the commands are all findable now

also updated readme with current info and description of localization

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,10 @@ An open source Python based app that replicates the functionality of the existin
 
 **Important Note:** tabcmd is a work in progress ("beta") which may be useful for test and development purposes, but is not yet recommended for production environments.
 
-
+tabcmd.exe is available at ____
 ## For user documentation see https://tableau.github.io/tabcmd/
```
Comment (mmuttreja-tableau) [inline README.md:11]: When I hit the download button on https://tableau.github.io/tabcmd/, it takes me to  https://codeload.github.com/tableau/tabcmd/zip/2.0.0 and I get a 404.

File: README.md [hunk 2]
```diff
@@ -23,43 +24,37 @@ An open source Python based app that replicates the functionality of the existin
...
-####To work with tabcmd, you need to have **Python 3.7+** installed.
-
 To install tabcmd, follow these steps:
+0. To work with tabcmd, you need to have **Python 3.7** or higher installed, and pip
+*see > https://pip.pypa.io/en/stable/installation/ *
+1. Clone the repo and install the app
+> $ git clone https://github.com/tableau/tabcmd.git
+> 
+> $ pip install .
+2. run tests
```
Comment (mmuttreja-tableau) [inline README.md:34]: Minor - Can we change `run tests` to `(optional) run tests` since people can use tabcmd as a user and they don't need to run tests.

**Comment by jacalata** [comment]
@smarvin here I made large changes to the Readme and split out a contributing.md, could you take a look?

**Comment by smarvin** [comment]
@jacalata , it looks quite but I'd like to make a proper editorial pass (just a comma here and there). I'm not smart enough to know how to do this in a pull request. Would it be easier just edit after your pull request is merged and I'll make edits and another pull request?

**Comment by jacalata** [comment]
> @jacalata , it looks quite good but I'd like to make a proper editorial pass (just a comma here and there). I'm not smart enough to know how to do this in a pull request. Would it be easier just edit after your pull request is merged and I'll make edits and another pull request?

I think it would be easier, yes.

**Pull request merged**